6.2.2000_runing.py

```diff
@@ -0,0 +1,16 @@
+from timeit import default_timer
+
+
+# define a function that get a function and her parameter as a parameter and check the time it takes to run
+def timer(func,*args ,**kwargs):
+    start = default_timer()
+    func(*args,**kwargs)
+    end = default_timer()
+    return end - start
+
+
+
+'''x =timer("Hi {name}".format, name="Bug")
+print(x)
+Y =timer(zip, [1, 2, 3], [4, 5, 6])
+print(Y)'''
```

Comment:
If you don't use it, remove it, and if you do use it Use _name==main_ - 
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/